wled00/util.cpp

```diff
@@ -795,6 +795,7 @@ String generateDeviceFingerprint() {
   #define BIT_WIDTH ADC_WIDTH_BIT_12
   #endif
   esp_adc_cal_characterize(ADC_UNIT_1, ADC_ATTEN_DB_11, BIT_WIDTH, 1100, &ch);
+  #undef BIT_WIDTH
```

Comment:
if you want to avoid name clashes, I would recommend to use ``constexpr``:
````C++
   constexpr auto myBIT_WIDTH = ADC_WIDTH_BIT_12;
   esp_adc_cal_characterize(ADC_UNIT_1, ADC_ATTEN_DB_11, myBIT_WIDTH, 1100, &ch);
````

Reply:
It's probably best to just commit the fix rather than comment

Reply:
right, i just did not want to interfere with your coding 😅 . I'll simply commit a fix.